examples/ecs/ambiguity_checker.rs

```diff
@@ -0,0 +1,63 @@
+use bevy::{ecs::schedule::ReportExecutionOrderAmbiguities, log::LogPlugin, prelude::*};
+
+fn main() {
+    App::new()
+        .add_plugin(LogPlugin)
+        .insert_resource(MyStartupResource(0))
+        // This resource allows to control how Ambiguity Checker will report unresolved ambiguities
+        // By default only a warning numbering the unresolved ambiguities count is show, but by
+        // explicitly setting it to verbose, a complete report is shown
+        .insert_resource(ReportExecutionOrderAmbiguities::verbose())
+        // startup_system_a and startup_system_b will both compete by the same resource. Since there is no ordering between
+        // both of them (like using .before or .after) there is no guarantee which one will take resource first.
+        // This ambiguity will be reported by Ambiguity Checker.
```

Comment:
```suggestion
        // `startup_system_a` and `startup_system_b` will both compete for the same resource. Since there is no ordering between
        // them (e.g., `.before()` or `.after()`), which one will run first is not deterministic.
        // This ambiguity will be reported by ambiguity checker.
```